zookeeper-server/src/main/java/org/apache/zookeeper/ServerAdminClient.java

```diff
@@ -52,7 +53,7 @@ public static void ruok(String host, int port) {
             byte[] resBytes = new byte[4];
 
             int rc = is.read(resBytes);
-            String retv = new String(resBytes);
+            String retv = new String(resBytes, StandardCharsets.UTF_8);
```

Comment:
These would be great to use static imports for the `UTF_8`, as was done in some of the other classes.